src/scripts/main.js

```diff
@@ -1,3 +1,316 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+const table = document.querySelector('table');
+const headers = table.querySelectorAll('th');
+const tBodies = table.querySelector('tbody');
+
+let selectedRow = null;
+let activeCell = null;
+
+const directions = Array.from(headers).map(function(header) {
+  return '';
+});
```

Comment:
```suggestion
const directions = Array.from(headers).map(() => '');
```
You can do it simpler